scripts/build-prep-1node.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
+
+# Clone, build and prepare to start a single-node cluster.
+#
+# Usage: ./build-prep-1node.sh [-dev]
+#
+#   -dev  development mode, don't build and install rpms
+#
+set -e -o pipefail
+
+dev_mode=
+
+[[ $1 == "-dev" ]] && dev_mode='yes'
+
+[[ -d cortx-motr ]] || {
+    git clone --recurse https://github.com/Seagate/cortx-motr.git &&
+        ln -s cortx-motr motr
+}
+
+cd motr
+echo 'Configure Motr...'
+[[ -f configure ]] && git clean -dfx
+./autogen.sh && ./configure --disable-expensive-checks
```

Comment:
Let's add an option "--with--user-mode-only", in that case Lnet and kernel modules are not needed.
May be an option can be added to this script to do it.

Reply:
Ok, will add.

Reply:
On another thought, let's do this after Motr is able to work with the default libfabric version maybe? Otherwise, how external users will be able to access our custom libfabric version?

Reply:
https://github.com/Seagate/cortx/releases/download/build-dependencies/isa-l-2.30.0-1.el7.x86_64.rpm
https://github.com/Seagate/cortx/releases/download/build-dependencies/libfabric-1.11.2-1.el7.x86_64.rpm
https://github.com/Seagate/cortx/releases/download/build-dependencies/libfabric-devel-1.11.2-1.el7.x86_64.rpm

All these packages are available for public use.

Reply:
Ok, will add downloading and installing these packages too...

Reply:
Added.